include/macros.h

```diff
@@ -161,4 +161,10 @@ extern GraphicsContext* __gfxCtx;
                        ((height)-1) << G_TEXTURE_IMAGE_FRAC);                                                          \
     } while (0)
 
+#ifdef __GNUC__
+#define ALIGNED8 __attribute__ ((aligned (8)))
+#else
+#define ALIGNED8
```

Comment:
so what's the point of this? planning ahead for gcc support?

Reply:
Yeah, this is because of a discussion we had on discord. The idea is to have better compatibility with gcc. ZAPD in the future will start using this macro for the extracted assets, so we will need it in the repo.